docs/consensus/security.md

```diff
@@ -13,4 +13,75 @@ keywords:
 - Constraining Simulation
 - Handling Equivocation
 - Detecting Long-Range Attacks -->
-For detailed security analysis please refer to our paper [Dilithium: A Proof-of-Archival-Storage Consensus Protocol for Subspace](https://github.com/subspace/consensus-v2-research-paper)
\ No newline at end of file
+
+A complex system like the Subspace protocol has many potential attack vectors, some are more general, blockchain related, 
+while other are focused on proof of capacity and specifically proof of archive storage, as used in Subspace.  
+
+This page attempts to give an overview of how such attacks are mitigated in the Subspace protocol. There are many things
+ to cover, so the reader should be aware that at the current state this page is not exhaustive. Furthermore, the 
+presentation mostly highlights the techniques used.
+For detailed security analysis please refer to our paper [Dilithium: A Proof-of-Archival-Storage Consensus Protocol for Subspace](https://github.com/subspace/consensus-v2-research-paper).
+
+# Security against general blockchain attacks
+
+## Grinding on block challenges
+
+To prevent grinding on block challenges, we use the proof-of-time outputs to draw unique challenges.
+
+Unlike in proof-of-work-based blockchains, where the "challenge" for block creation comes from the previous block, in 
+Subspace this is not the case, as changing the block content does not affect the proof of space validity. Instead, 
```

Comment:
The sentence structure is awkward, maybe something like:

In Subspace, unlike in proof-of-work-based blockchains where the "challenge" for block creation comes from the previous block, changing the block content does not affect the proof of space validity.

Reply:
Better now? I had some issues with adopting your suggestion.